packages/react-renderer-demo/src/app/src/doc-components/mui-slider.md

```diff
@@ -0,0 +1,19 @@
+Please check the [original component](https://material-ui.com/components/slider/#slider), it accepts the same props.
+
+**Props**
+
+Slider is wrapped in a form group, so it accepts all [form group props](/renderer/component-api#formgroupwrappedcomponents).
+
+|Props|Type|Description|
+|-----|----|-----------|
+|after|node|Node rendered after the slider|
+|before|node|Node rendered before the slider|
+|FormFieldGridProps|object|Props passed to FormFieldGrid component|
```

Comment:
Can we add links to the corresponding API of the MUI component? The rest of the MUI components does have this link.

Reply:
done